### PR TITLE
Flask db

### PR DESCRIPTION
**Pull request by rohan-suresh**
added get and post endpoint for database that interacts with ember client

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
 
 app = Flask(__name__)
 CORS(app)
 
+with open('config_file.json') as config_file:
+	config = json.load(config_file)
+
+conn = mdb.connect(host=config['host'], user=config['user'], passwd=config['password'], db=config['db'])
+
+
```
Comment (adrianlizarraga) [inline app.py]: Remove this extra empty line

File: app.py [hunk 2]
```diff
@@ -13,10 +23,31 @@ def hello_world():
...
-@app.route('/getData', methods = ['GET'])
-def getData():
-	return json.dumps({"success": True,
-  "events": [ {"chapter_number": 1, "section_number": 1, "user_id": 333, "date": datetime.datetime.now().isoformat()}, {"chapter_number": 1, "section_number": 1, "user_id": 444}]})
+def getEvents(zybook_code, user_id):
+	query = "SELECT event FROM user WHERE zybook_code=" + "'" + zybook_code + "'"
+	cur.execute(query)
+	rows = cur.fetchall()
+	user_rows = []
+	print(user_id)
```
Comment (adrianlizarraga) [inline app.py]: Remove print statements

File: app.py [hunk 3]
```diff
@@ -13,10 +23,31 @@ def hello_world():
 def test():
     return 'Hello, test!'
 
-@app.route('/getData', methods = ['GET'])
-def getData():
-	return json.dumps({"success": True,
-  "events": [ {"chapter_number": 1, "section_number": 1, "user_id": 333, "date": datetime.datetime.now().isoformat()}, {"chapter_number": 1, "section_number": 1, "user_id": 444}]})
+def getEvents(zybook_code, user_id):
```
Comment (adrianlizarraga) [inline app.py]: Our convention for Ember code is to use camelCase. However, for _python_ code we use lower case and dashes for all variables and method names. Ex: `get_events`. Let's rename variables and methods to use this style.

File: app.py [hunk 4]
```diff
@@ -13,10 +23,31 @@ def hello_world():
...
+	user_rows = []
+	print(user_id)
+	for elem in rows:
+		if json.loads(elem[0])['event']['user_id'] == int(user_id):
+			user_rows.append(json.loads(elem[0])['event'])
+
+	return user_rows
+
+@app.route('/zybook/<zybook_code>/events/<user_id>', methods = ['GET'])
+def getData(zybook_code, user_id):
```
Comment (adrianlizarraga) [inline app.py]: Let's rename this method to something more descriptive like `get_user_zybook_events`

File: app.py [hunk 5]
```diff
@@ -13,10 +23,31 @@ def hello_world():
...
+	user_rows = getEvents(zybook_code, user_id)
+	return json.dumps({'success': True, 'events': user_rows})
+
+@app.route('/zybook/<zybook_code>/event', methods = ['POST'])
+def postData(zybook_code):
```
Comment (adrianlizarraga) [inline app.py]: Same here. I would recommend renaming to something like `record_zybook_event`

File: app.py [hunk 6]
```diff
@@ -13,10 +23,31 @@ def hello_world():
...
+	cur.execute("INSERT INTO user(zybook_code, event) VALUES(%s, %s)", (zybook_code, json.dumps({'event': {'chapter_number': request.json.get('chapterNum'), 'section_number': request.json.get('sectionNum'), 'user_id': request.json.get('user_id'), 'date': datetime.datetime.now().isoformat()}}))) #
+	#datetime.datetime.now().isoformat()
```
Comment (adrianlizarraga) [inline app.py]: Let's remove commented lines from code.

**Review by adrianlizarraga — CHANGES_REQUESTED** [review]
@rohan-suresh Left you a few comments. Looks good!

File: app.py (hunk 1 above)
Comment (rohan-suresh) [reply app.py]: resolved

File: app.py (hunk 3 above)
Comment (rohan-suresh) [reply app.py]: resolved

File: app.py (hunk 2 above)
Comment (rohan-suresh) [reply app.py]: resolved

File: app.py (hunk 4 above)
Comment (rohan-suresh) [reply app.py]: resolved

File: app.py (hunk 5 above)
Comment (rohan-suresh) [reply app.py]: resolved

File: app.py (hunk 6 above)
Comment (rohan-suresh) [reply app.py]: resolved

**Review by adrianlizarraga — APPROVED** [review]

**Pull request merged**